tests/conditional_formatting/conditional_formatting_panel_component.test.ts

```diff
@@ -440,6 +440,54 @@ describe("UI of conditional formats", () => {
       );
     });
 
+    test("Can remove a valid, invalid or empty range", async () => {
+      await click(fixture, selectors.buttonAdd);
```

Comment:
Could you add a similar global test for all selection inputs in `selection_input_component.test.ts`?